ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -115,3 +126,51 @@ If.Capture {
 		cset "name='Capture Switch' on"
 	]
 }
+
+Include.hdmi-pcm.File "/common/pcm/hdmi.conf"
+
+If.Hdmi3-iec61937 {
+	Condition {
+		Type RegexMatch
+		Regex "((^|,)[3](,|$))"
+		String "${var:Iec61937Pcms1}"
+	}
+	True {
+		Macro.hdmi3.HdmiPCM { Device 3 Index 0 }
+		Define.hdmi_iec61937 "yes"
+	}
+}
+
+If.Hdmi4-iec61937 {
+	Condition {
+		Type RegexMatch
+		Regex "((^|,)[4](,|$))"
+		String "${var:Iec61937Pcms1}"
+	}
+	True {
+		Macro.hdmi4.HdmiPCM { Device 4 Index 1 }
+		Define.hdmi_iec61937 "yes"
+	}
+}
+
+If.Hdmi5-iec61937 {
+	Condition {
+		Type RegexMatch
+		Regex "((^|,)[5](,|$))"
+		String "${var:Iec61937Pcms1}"
+	}
+	True {
+		Macro.hdmi5.HdmiPCM { Device 5 Index 2 }
+		Define.hdmi_iec61937 "yes"
+	}
+}
+
+If.HdmiIec61937 {
+	Condition {
```

Comment:
Probably last thing I forgot to note in the first review - can we use RegexMatch here using the `${var:Iec61937Pcms1}` string instead a temporary variable?

Like `Regex "((^|,)[345](,|$))" ?

It saves some lines.